index.js

```diff
@@ -146,6 +196,32 @@ class SocketValidationError extends Error {
   }
 }
 
+/**
+ * @param {object} parsedV1Content
+ * @returns {Promise<SocketYml | null>}
+ */
+async function parseV1SocketConfig (parsedV1Content) {
+  if (!validateV1(parsedV1Content)) {
+    return null
+  }
+
+  const v2 = {
+    version: 2,
+    projectIgnorePaths: parsedV1Content?.ignore ?? [],
+    issueRules: parsedV1Content?.issues ?? {},
+    githubApp: {
+      enabled: parsedV1Content?.enabled,
+      pullRequestAlertsEnabled: parsedV1Content?.pullRequestAlertsEnabled,
+      projectReportsEnabled: parsedV1Content?.projectReportsEnabled
+    }
+  }
+
+  if (!validate(v2)) {
+    return null
+  }
+  return v2
```

Comment:
```suggestion
  if (validate(v2)) {
    return v2
  }
```

Lets skip `null` and just use `undefined`